packages/scrapers-v2/src/catalog_scraper.ts

```diff
@@ -0,0 +1,377 @@
+import { assertUnreachable } from "@graduate/common";
+import axios from "axios";
+import * as cheerio from "cheerio";
+import {
+  COURSE_REGEX,
+  RANGE_1_REGEX,
+  RANGE_2_REGEX,
+  RANGE_3_PARSE_REGEX,
+  RANGE_3_REGEX,
+} from "./constants";
+import {
+  CourseRow,
+  HDocument,
+  HRow,
+  HRowType,
+  HSection,
+  MultiCourseRow,
+  RangeRow,
+  TextRow,
+} from "./types";
+
+const loadCatalogHTML = async (url: string): Promise<CheerioStatic> => {
+  try {
+    const { data } = await axios.get(url);
+    return cheerio.load(data);
+  } catch (error) {
+    throw error;
```

Comment:
we should write a utility for logging errors

Reply:
coming in the next part when doing audit stuff